ci/fix_gitignore.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+# shellcheck disable=SC2012
+
+set -euo pipefail
+# shellcheck disable=SC1091
+source ci/ci_helpers.sh
+
+ls -1 src/test/regress/input | sed -E "s/(.*)\.source/\/\1.sql/" > src/test/regress/sql/.gitignore
```

Comment:
usually when there is regex involved, I found that having a comment helps, because it is a bit hard to read regexes in general. Like having a comment that explains this line at a high level could be useful, what do you think?